### PR TITLE
Fixes #21459 - '--state' hidden from foreman-tasks-fetch-tasks-status

### PR DESCRIPTION
**Pull request by kgaikwad**

### CODE REVIEW

**Comment by kgaikwad** [comment]
Job failed for ruby 2.3.3 due to rubocop errors which are fixed in [PR-116](https://github.com/theforeman/foreman_maintain/pull/116)

**Comment by kgaikwad** [comment]
Updated the PR. Now, ready for review.

File: definitions/procedures/foreman_tasks/fetch_tasks_status.rb [hunk 1]
```diff
@@ -3,7 +3,7 @@ class FetchTasksStatus < ForemanMaintain::Procedure
     metadata do
       for_feature :foreman_tasks
       description 'Fetch tasks status and wait till they finish'
-      param :state, :required => true
+      param :state, 'Specify state of tasks. Either running or paused', :required => true
```
Comment (swapab) [inline definitions/procedures/foreman_tasks/fetch_tasks_status.rb]: Emphasize states by enclosing them in quotes('running' or 'paused')

**Comment by swapab** [comment]
LGTM Thanks @kgaikwad

**Pull request merged**